cognite_toolkit/_repo_files/AzureDevOps/.devops/deploy-pipeline.yml

```diff
@@ -0,0 +1,36 @@
+trigger:
+  branches:
+    include:
+      - main
+
+jobs:
+- job: DeployModules
+  displayName: 'Deploy to Dev'
+  pool:
+    vmImage: 'ubuntu-latest'
+  container:
+    image: cognite/toolkit:0.3.1
+  variables: 
+    - group: 'dev-toolkit-credentials'
+  steps:
+    - checkout: self
+      clean: true
+
+    # Uncomment the following lines to enable statistics
+    # - script:
+    #     cdf collect opt-in
+    #   displayName: 'Enable statistics'
+    
+    - script: |
+        cdf build
+      displayName: 'Build the modules'
```

Comment:
Need environment variables in build as well. Is there a way of setting them on the job level?

Reply:
no, at least not one that becomes cleaner, so I just duplicated them